ci/laravel.yml

```
@@ -12,6 +12,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+    - uses: shivammathur/setup-php@v2
```

```suggestion
    - uses: shivammathur/setup-php@b7d1d9c9a92d8d8463ce36d7f60da34d461724f8
```

Let's pin this to the 40 char sha for this repo as per the guidelines.